templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} Manufacturer</h1>
+  <form action="" method="post" novalidate>
+  {% csrf_token %}
+  {{ form|crispy }}
+  <input type="submit" value="Submit" class="btn btn-primary">
+  </form>
```

Comment:
According to the checklist, you should add a 'Cancel' button apart from the 'Submit' one. The 'Cancel' button should lead to the previous page the user was on.